MAUI/Popup/popup-size.md

```diff
@@ -0,0 +1,223 @@
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PopupMaui.FullScreenPage"
+             xmlns:local="clr-namespace:PopupMaui"
+             xmlns:inputLayout="clr-namespace:Syncfusion.Maui.Core;assembly=Syncfusion.Maui.Core"
+             xmlns:popup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             >
+    <ContentPage.Content>
+        <StackLayout Padding="20">
+            <Button x:Name="show" Text="Show Popup" Clicked="show_Clicked"/>
+            <popup:SfPopup x:Name="popup" AppearanceMode="TwoButton" AcceptButtonText="SAVE"  DeclineButtonText="CANCEL" ShowCloseButton="True" ShowFooter="True">
+                <popup:SfPopup.HeaderTemplate>
+                    <DataTemplate>
+                        <Label Text="ADD EVENT" VerticalTextAlignment="Center" FontSize="18" HorizontalTextAlignment="Start" FontAttributes="Bold"/>
+                    </DataTemplate>
+                </popup:SfPopup.HeaderTemplate>
+                <popup:SfPopup.ContentTemplate>
+                .......
+                </popup:SfPopup.ContentTemplate>
+                <popup:SfPopup.PopupStyle>
+                    <popup:PopupStyle CornerRadius="0"/>
+                </popup:SfPopup.PopupStyle>
+            </popup:SfPopup>
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="MainPage.xaml.cs"  hl_lines="15 16 19" %}
+using Syncfusion.Maui.Popup;
+
+namespace PopupMaui
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+        }
+
+        private void show_Clicked(object sender, EventArgs e)
+        {
+            // Shows SfPopup in full width and height of the screen using the SfPopup.IsFullScreen property.
+            popup.IsFullScreen = true;
+            popup.IsOpen = true;
+            
+            // Shows SfPopup in full width and height of the screen using the SfPopup.Show(bool) method.
+            // popup.Show(true);
+        }
+    }
+}
+{% endhighlight %}
+
+Executing the above codes renders the following output in windows.
+
+![.NET MAUI Popup with Full Screen](Images/popup-size//maui-popup-with-fullscreen.png)
+
+
+## Auto-size Popup
+
+The `SfPopup` can auto-size the popup view based on the contents loaded inside its `SfPopup.ContentTemplate` property using the `SfPopup.AutoSizeMode` property. The default value is `AutoSizeMode.None`. You can choose to auto-size the Popup in height, width or in both height and width of its contents. By default, the `HeightRequest` and `WidthRequest` set to the `SfPopup` or the views loaded inside the template is given higher priority than the `AutoSizeMode`.
+
+In the following code snippet the Popup is auto-sized in height based on the content loaded inside the `SfPopup.ContentTemplate` property.

```

Comment:
In the following code sample, the Popup is auto-sized in the height based on the content loaded inside the `SfPopup.ContentTemplate` property.

Reply:
Addressed the concerns provided.